docs/tutorials.rst

```diff
@@ -3,6 +3,14 @@
 Tutorials
 ---------
 
+.. note::
+
+     **We're building the next version of Renku!** If you're looking for a tutorial for Renku 2.0,
+     please see the Renku 2.0 documentation on our `Community Portal
+     <https://renku.notion.site/Renku-Community-Portal-2a154d7d30b24ab8a5968c60c2592d87>`_ instead.
+     To learn more about the big changes coming in Renku, check out our `blog post
+     <https://blog.renkulab.io/renku-2/>`_.
+
```

Comment:
should we be more explicit here and say something like "The tutorial linked below is outdated and refers to the legacy version of Renku that we will no longer support." ?

Reply:
Good idea!

I feel like "no longer support" might scare current users more than it needs to. I chose "no longer under active development" instead.